### DISCUSSION TITLE
Function to customize msg type

### DISCUSSION THREAD

**Discussion by Emzi-beta** [ika-rwth-aachen/mqtt_client#54] [General]
According to the document, mqtt_client_interface/msg/RosMsgType.msg seems to be used to get the metadata for Rostopic, but is there no function customizing msg type such as robotState{abc', robotNm : "agv1", currentCoord : {x : 0.1, y : 0.2}}?

**Comment by lreiher** [comment]
Sorry, I seem to have overlooked your question. Can you clarify, I don't really understand the question?
